packages/toolkit/.changes/next-release/Test-de189b97-989c-4c35-9a72-c874bb72cc92.json

```diff
@@ -0,0 +1,4 @@
+{
+	"type": "Test",
+	"description": "SAM: update sync command tests"
```

Comment:
Only user-facing changes need a changelog. Please read the [changelog guidelines](https://github.com/aws/aws-toolkit-vscode/blob/master/CONTRIBUTING.md#changelog).

Reply:
Actually, this PR is about refactoring test for sync command. A change log is not needed.

Reply:
Remove change log.